src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -15,32 +15,76 @@ export interface MinigraphOptions {
   borderColor?: string;
 }
 
+enum Zoom {
+  In,
+  Out
+}
+
 export const defaultMinigraphOptions: MinigraphOptions = {
   backgroundColor: NodePlotColor,
   borderColor: NodePlotColor
 };
 
 let stepY = 5;
 
-export const DataflowNodePlot = (props: NodePlotProps) => {
+export const DataflowNodePlot: React.FC<INodePlotProps> = (props) => {
   if (!props.display) return null;
 
+  const handleClickOffset = (zoomDir: Zoom) => {
+    const max = props.data.data.tickMax || props.data.data.dsMax;
+    const min = props.data.data.tickMin || props.data.data.dsMin;
```

Comment:
```suggestion
    const max = props.data.data.tickMax ?? props.data.data.dsMax;
    const min = props.data.data.tickMin ?? props.data.data.dsMin;
```
`||` would return `dsMax` if `tickMax` is `0`.

Reply:
true, good catch.